src/actions/orders/create.ts

```diff
@@ -0,0 +1,102 @@
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { transaction } from 'objection';
+import Order from '../../models/Order';
+import OrderItem from '../../models/OrderItem';
+import Product from '../../models/Product';
+import { OrderStatus } from '../../models/Order';
+import { ValidationError } from 'objection';
+

```

Comment:
é interessante importar todas as dependências externas antes das internas, assim vc nao tem problemas com pacotes faltando ao rodar seu próprio código:

```suggestion
import { FastifyReply, FastifyRequest } from 'fastify';
import { transaction, ValidationError } from 'objection';
import Order from '../../models/Order';
import OrderItem from '../../models/OrderItem';
import Product from '../../models/Product';
import { OrderStatus } from '../../models/Order';


```

pode ser que isso seja um problema antigo que já foi solucionado, mas fica como boa prática 😄